tests/backup/backup_helper.go

```diff
@@ -3393,3 +3394,13 @@ func GenerateRandomLabelsWithMaxChar(number int, charLimit int) map[string]strin
 	}
 	return labels
 }
+
+// GetCustomBucketName creates a custom bucket and returns name
+func GetCustomBucketName(provider string, testName string) string {
```

Comment:
How will you handle if the provider is `nfs` ?

Reply:
Didn't get you